pyQuARC/code/custom_validator.py

```diff
@@ -31,8 +33,8 @@ def ends_at_present_flag_presence_check(
         ends_at_present_flag, ending_date_time, collection_state
     ):
         valid = True
-        if not ends_at_present_flag:
-            valid = ending_date_time or collection_state == "COMPLETE"
+        if ends_at_present_flag == None:
+            valid = not not (ending_date_time) and collection_state == "COMPLETE"
```

Comment:
`not not`? Trying to convert to boolean?

Reply:
@spa0002 ?

Reply:
Yeah, so that None type would be False.

Reply:
see comment above.